### DISCUSSION TITLE
tampa,florida

### DISCUSSION THREAD

**Discussion by pallavisharma15** [OvertureMaps/data#166] [Discussion]
### Category Feedback

Add details 
 - Expected behaviour you didn't see 
 - Unexpected behaviour 
 - suggested functionality 
 - Steps to reproduce the problem

### Dependency with other categories, if any.

Address, Schema

**Comment by jwass** [comment]
Hi @pallavisharma15 looks like this an #165 were both submitted without any information.

I'll close both but feel free to reopen with more details.
